gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java

```diff
@@ -155,42 +181,6 @@ private static void applyToJavaProject(Project project) {
                                 errorProneOptions.check("PreconditionsConstantMessage", CheckSeverity.OFF);
                             }));
         });
-
-        project.getTasks().withType(JavaCompile.class).configureEach(javaCompile -> {
-            ((ExtensionAware) javaCompile.getOptions())
-                    .getExtensions()
-                    .configure(ErrorProneOptions.class, errorProneOptions -> {
-                        // Relax some checks for test code
-                        if (errorProneOptions.isCompilingTestOnlyCode()) {
-                            errorProneOptions.check("UnnecessaryLambda", CheckSeverity.OFF);
-                        }
-                    });
-        });
```

Comment:
this block was pretty much an exact duplicate of a block above, so I moved this logic further down.